### DISCUSSION TITLE
CGAL linking with the new cmake?

### DISCUSSION THREAD

**Discussion by avaxman** [libigl/libigl#1996] [Q&A]
I got lost on how to add CGAL with the new Cmake. Is there a documentation/example of this? it's a bit difficult to follow what the tutorial does.

**Comment by alecjacobson** [comment]
Checkout the `cgal` branch of the libigl-example-project. From your project's CMakeListst.txt it's very easy to add cgal: https://github.com/libigl/libigl-example-project/tree/cgal
